dataavailability/dataavailability.go

```diff
@@ -113,7 +113,7 @@ func (d *DataAvailability) trustedSequencerData(batchNums []uint64, expectedHash
 		return nil, fmt.Errorf("invalid arguments, len of batch numbers does not equal length of expected hashes: %d != %d",
 			len(batchNums), len(expectedHashes))
 	}
-	nums := make([]*big.Int, 0, len(batchNums))
```

Comment:
nit: why this change?

Reply:
[c95ebf6](https://github.com/Layr-Labs/cdk-validium-node/pull/1/commits/c95ebf6fbc366991125f81567ee8c987b440b4f4)